Neos.ContentRepository.Core/Classes/Feature/NodeMove/NodeMove.php

```diff
@@ -149,7 +150,8 @@ private function handleMoveNodeAggregate(
             $originNodeMoveMappings[] = new OriginNodeMoveMapping(
                 $movedNodeOrigin,
                 $this->resolveCoverageNodeMoveMappings(
-                    $contentStreamId,
+                    // todo UNSAFE as override via ContentStreamIdOverride will NOT be taken into account!!!
+                    $command->workspaceName,
```

Comment:
this MUST not be used as a custom repoint via `ContentStreamIdOverride` would be ignored.
We  possibly have to pass both as touple and create a `new ContentGraph` by hand.